walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/rawrequests/VerifiedIdOpenIdJwtRawRequestTest.kt

```diff
@@ -98,11 +101,15 @@ class VerifiedIdOpenIdJwtRawRequestTest {
 
     @Test
     fun mapOpenIdJwtRawRequest_mapPresentationRequestWithLogoToRequestContent_ReturnsRequestContent() {
+        // Arrange
+        every { mockPresentationRequest.content.redirectUrl } returns expectedCallbackUrl
+        every { mockPresentationRequest.content.state } returns expectedRequestState
+
         // Act
-        val actualResult = verifiedIdOpenIdJwtRawRequest.mapToRequestContent()
+        val actualResult = verifiedIdOpenIdJwtRawRequest.mapToPresentationRequestContent()
 
         // Assert
-        assertThat(actualResult).isInstanceOf(VerifiedIdRequestContent::class.java)
+        assertThat(actualResult).isInstanceOf(com.microsoft.walletlibrary.requests.PresentationRequestContent::class.java)
         assertThat(actualResult.requesterStyle.requester).isEqualTo(expectedEntityName)
```

Comment:
Is there a test that checks to see that the actualResult.callback and state are equal to the mock values?